### ISSUE TITLE
[ES] Not detecting temperature in "37ºC"

### ISSUE THREAD

**Issue by ADD-eNavarro** [microsoft/Recognizers-Text#2877]
**Describe the bug**
I run all recognizers over "Temperatura 37ºC". 0 results in Temperature, but also in number.

**To Reproduce**
This simple code shows "Found 0 results".

```
            string txt = "Temperatura 37ºC";
            var results = Helpers.ParseAll(txt, "es-es").ToList(); // Get ALL recognizers on the text

            Console.WriteLine("Found {0} results.", results.Count);
```

**Expected behavior**
Since [number]+ºC is a standard way of writing Celsius degrees, it should be recognized.

**Platform (please complete the following information):**
 - Platform:  .NET Core 3.1
 - Environment: Console app
 - Version of package: 1.8.2

**Comment by tellarin** [comment]
Technically this is not a correct case as that's the upperscript o, not the degree symbol. But we do understand that in informal cases this can be a common usage (in PT too). It should be easy to extend support if it doesn't introduce any negative side effect.

**Comment by ADD-eNavarro** [comment]
Yes indeed, I'm working with lots of real world cases and "º" (upperscript) is almost the only one used out there, since "°" (ALT+0176) is way too unknown to people. So yes, extending the support to upperscript will make it much more usable.
Thank you.
